src/pages/Profile/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import { Result } from "../../components";
+
+const Profile = () => {
+  //   const [user, setUser] = useState("");
+  let user = "humza1997";
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let input = e.target.value;
+    setUser(input);
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <label>User</label>
+        <input id='User' type='text' value={user} onChange={(e) => setUser(e.target.value)} />
+        <input type='submit' value='Search' />
+      </form>
+      <Result user={user} />
+      <p>Heres your profile</p>;
+    </div>
+  );
+};
```

Comment:
```jsx
const Profile = () => {
	const [input, setInput] = useState('');
	const [user, setUser] = useState('');
	// let user = "humza1997";

	const handleSubmit = (e) => {
		e.preventDefault();
		setUser(input);
		// setInput('') // to clear input on form submit (optional)
	};

	const onChange = (e) => setInput(e.target.value);

	return (
		<div>
			<form onSubmit={handleSubmit}>
				<label htmlFor='User'>User</label>
				<input name='User' id='User' type='text' value={input} onChange={onChange} />
				<input type='submit' value='Search' />
			</form>
			<Result user={user} />
			<p>Heres your profile</p>;
		</div>
	);
};
```

Reply:
You need one state to manage the input `onChange` and `value` attributes and one to store the value from form submission.
In you `handleSubmit` function (line 10) `e.target.value` is undefined as the form has no `value` rather than the intended value from your input field.

Reply:
Also added missing attributes to label and text input